### PR TITLE
VM/sac fix and cleanup

### PR DESCRIPTION
**Pull request by ViktorM**
- SAC fixes and improvements
- Better SAC configs
- Better envpool benchmarking configs

### CODE REVIEW

**Review by Denys88 — APPROVED** [review]

File: rl_games/configs/atari/ppo_breakout.yaml [hunk 1]
```diff
@@ -39,41 +38,45 @@ params:
...
-    normalize_value: True
-    normalize_input: False
-    max_epochs: 3000
+    max_epochs: 1500
+
     env_config:
       skip: 4
       name: BreakoutNoFrameskip-v4
-      episode_life: True
+      episodic_life: True
```
Comment (Denys88) [inline rl_games/configs/atari/ppo_breakout.yaml:76]: for ray this parameter is called: episode_life

File: rl_games/configs/atari/ppo_breakout.yaml [hunk 2]
```diff
@@ -39,41 +38,45 @@ params:
...
+      has_lives: True
```
Comment (Denys88) [inline rl_games/configs/atari/ppo_breakout.yaml:77]: dont need this parameteer for ray

File: rl_games/configs/atari/ppo_pong.yaml [hunk 3]
```diff
@@ -6,22 +6,22 @@ params:
...
       discrete:
 
     cnn:
+      #permute_input: False
       type: conv2d
       activation: elu
       initializer:
-        name: glorot_normal_initializer
-        gain: 1.4142 
+        name: default
```
Comment (Denys88) [inline rl_games/configs/atari/ppo_pong.yaml:20]: are you sure default is better than glorot normal?

File: rl_games/configs/atari/ppo_breakout.yaml (hunk 1 above)
Comment (ViktorM) [reply rl_games/configs/atari/ppo_breakout.yaml:76]: Are you serious?! Why don't they have the same name?

**Review by Denys88 — APPROVED** [review]

**Pull request merged**